js/examples/notebooks/deno_datasets_and_experiments_quickstart.ipynb

```diff
@@ -0,0 +1,443 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<center>\n",
+    "    <p style=\"text-align:center\">\n",
+    "        <img alt=\"phoenix logo\" src=\"https://raw.githubusercontent.com/Arize-ai/phoenix-assets/9e6101d95936f4bd4d390efc9ce646dc6937fb2d/images/socal/github-large-banner-phoenix.jpg\" width=\"1000\"/>\n",
+    "        <br>\n",
+    "        <br>\n",
+    "        <a href=\"https://docs.arize.com/phoenix/\">Docs</a>\n",
+    "        |\n",
+    "        <a href=\"https://github.com/Arize-ai/phoenix\">GitHub</a>\n",
+    "        |\n",
+    "        <a href=\"https://arize-ai.slack.com/join/shared_invite/zt-2w57bhem8-hq24MB6u7yE_ZF_ilOYSBw#/shared-invite/email\">Community</a>\n",
+    "    </p>\n",
+    "</center>\n",
+    "<h1 align=\"center\">Quickstart: Datasets and Experiments in Deno</h1>\n",
+    "\n",
+    "Phoenix helps you run experiments over your AI and LLM applications to evaluate and iteratively improve their performance. This quickstart shows you how to get up and running quickly with the JavaScript SDK in a Deno environment."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "Let's start by importing the necessary packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import { createClient } from \"npm:@arizeai/phoenix-client@latest\";\n",
+    "import {runExperiment, asEvaluator, evaluateExperiment} from \"npm:@arizeai/phoenix-client@latest/experiments\";\n",
+    "import { createDataset } from \"npm:@arizeai/phoenix-client@latest/datasets\";\n",
+    "import { OpenAI } from \"npm:openai\";"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Set up your OpenAI API key."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "const openaiApiKey = prompt(\"Enter your OpenAI API key:\");\n",
+    "\n",
+    "if (!openaiApiKey) {\n",
+    "  console.error('Please enter your OpenAI API key to continue');\n",
+    "  Deno.exit(1);\n",
+    "}\n",
+    "\n",
+    "const openai = new OpenAI({\n",
+    "  apiKey: openaiApiKey,\n",
+    "});"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Initialize the Phoenix client."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "const client = createClient();\n",
+    "console.log('Phoenix client initialized. Access Phoenix UI at http://localhost:6006');"
```

Comment:
We may want to make it clear that this will not start phoenix for you, and you should get it running in some way. Maybe link to the docs?

https://docs.arize.com/phoenix/self-hosting/deployment-options/docker#docker